### PR TITLE
Fixed: Joystick labels on cleanup undo are `wire_joystick` not specif…

### PR DESCRIPTION
**Pull request by dvdvideo1234**
…ic to multy

### CODE REVIEW

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua [hunk 1]
```diff
@@ -4,7 +4,7 @@ local gsToolLimits = gsToolModeOP:gsub("_multi", "").."s"
 local gsSentClasMK = "gmod_"..gsToolModeOP
 local MappingFxUID = "abcdefghijklmnopqrstuvwxyz1234567890ABCDEFGHIJKLMNOPQRSTUVWXYZ"
 
-TOOL.Tab        = "Wire"
+TOOL.Tab        = gsToolModeOP:match("%w+"):gsub("^%l", string.upper)
```
Comment (MattJeanes) [inline addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua:7]: This seems unnecessarily complicated - can we just leave this at "Wire", same for the tool name below actually please

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua [hunk 2]
```diff
@@ -41,10 +41,10 @@ if CLIENT then
   language.Add( "tool."..gsToolModeOP..".analog_con"     , "Analog input" )
   language.Add( "tool."..gsToolModeOP..".config"         , "Click this button to open the joystick configuration. You can also right click on the world" )
   language.Add( "tool."..gsToolModeOP..".config_con"     , "Joystick Configuration" )
-  language.Add( "undone_"..gsToolModeOP, "Undone Wire Joystick Multi" )
-  language.Add( "sboxlimit_"..gsToolLimits, "You've hit the Joystick Multi limit!" )
-  language.Add( "cleanup_" .. gsToolLimits, "Wire Joystick Multi chips " )
-  language.Add( "cleaned_" .. gsToolLimits, "Cleaned up all Joystick Multi chips" )
+  language.Add( "undone_"..gsToolModeOP                  , "Undone Wire Joystick Multi !" )
```
Comment (MattJeanes) [inline addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua:44]: Why is there a space at end before the exclamation mark?

**Review by MattJeanes — CHANGES_REQUESTED** [review]

**Comment by dvdvideo1234** [comment]
@MattJeanes 

1. The idea is to be taken from the file name starting with `wire_` but. Ok 😊
2. Gotta remove it. Thank you.

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua (hunk 1 above)
Comment (dvdvideo1234) [reply addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua]: Reverted in d1a97f191828337d3eb16f73ce41a6784df705a1

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua (hunk 2 above)
Comment (dvdvideo1234) [reply addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua]: Reverted in d1a97f191828337d3eb16f73ce41a6784df705a1

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (dvdvideo1234) [reply addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick_multi.lua]: Reverted in d1a97f1

**Review by MattJeanes — APPROVED** [review]
Looks good, thank you!

**Pull request merged**